app/models/metadata_presenter/component.rb

```diff
@@ -10,6 +10,17 @@ class MetadataPresenter::Component < MetadataPresenter::Metadata
   # Threshold percentage at which the remaining count is shown
   VALIDATION_STRING_LENGTH_THRESHOLD = 75
 
+  # Overriding here because autocomplete component's items property is non interactable
+  # in the Editor therefore it does not need to exist in the data-fb-content-data
+  # attribute on the page. The data-fb-content-data attribute is what is used to
+  # update user interactable elements such as the component question or page section
+  # heading etc.
+  def to_json(*_args)
+    return super unless autocomplete?
+
+    JSON.parse(super).to_json(except: 'items')
```

Comment:
This is definitely cleaner!  Although I find it strange that you can't just do `super(except: 'items')` 🤷‍♂️